src/components/Footer/footer.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+const Footer: React.FC = () => {
+  type FooterList = {
+    id: number;
+    title: string;
+  };
```

Comment:
Please move this outside the component and the alias looks incorrect, it should be `FooterItem`